XamTwitch/XamTwitch/Services/TwitchAuthHttpService.cs

```diff
@@ -0,0 +1,28 @@
+﻿using System;
+using System.Net.Http;
+using System.Net.Http.Headers;
+using System.Threading.Tasks;
+using Newtonsoft.Json;
+using XamTwitch.Models;
+
+namespace XamTwitch.Services
+{
+    public class TwitchAuthHttpService
```

Comment:
Is it possible to inherit from BaseHttpService? It seems that the base URL is the same so we can also make this to be a part of `TwitchHttpService`.

https://github.com/xamcat/xam-twitch/blob/episode4a/XamTwitch/XamTwitch/Services/TwitchHttpService.cs#L15

Reply:
yeah I was going to refactor this in the next episode, didn't want to do the work offline. Unless we're all okay with me doing the refactor right now?

Reply:
but also, this uses a different Auth Header based Httpclient which can't directly work with our BaseHttpService. It will need some interesting fix to make it work with our current Base service 🤔

Reply:
It makes even more sense not to refactor it now but keep it for the live stream, to make sure we have everything covered by the live stream video :+1:

as for header, I guess we can set it as we do for the ClientId and as long as these calls should be authenticated we can totally keep them in different classes (services) as you currently implemented:

https://github.com/xamcat/xam-twitch/blob/episode4a/XamTwitch/XamTwitch/Services/TwitchHttpService.cs#L19

Reply:
Yep I was planning to use that for the refactoring! 😄  I'll merge this and next episode we'll get this cleaned up. 👍